### DISCUSSION TITLE
Add an expand all/collapse all button for categories (and structures) in the admin panel

### DISCUSSION THREAD

**Discussion by cp-gregj** [craftcms/cms#8951] [Ideas] [answered]
One of our clients currently uses a category set to organise part of their website, but they now have quite a lot of categories (thousands total, but only 4 top level) and they're finding managing this a bit frustrating as they have to close a lot of categories to get to the one that the ones that they actually want open. Having the ability to expand all and collapse all categories would make editing this much easier when there are a lot of categories.

Could be good for multi-level structures too.

**Comment by hiasl** [comment]
Just wanted to request this for **multi-level structures** and found this request. 
+1 from our company / clients

**Comment by dennisfrank** [comment]
Maybe OS like shortcuts could work? In many macOS apps you can expand/collapse nested lists by with <kbd>option</kbd> + clicking a list item arrow. 

Finder.app for example:

![Screenshot 2021-11-12_18-45-00 2021-11-12 18_48_23](https://user-images.githubusercontent.com/65968/141511875-2c505b30-32f8-422a-af48-b0c8686f2706.gif)

Not sure, if Windows offers a similar behavior.

**Comment by brandonkelly** [comment, marked as answer]
Closing this in favor of #8576.
